scss/_popover.scss

```diff
@@ -57,18 +58,17 @@
 
     .arrow::before,
     .arrow::after {
+      top: 100%;
```

Comment:
Please do not use any CSS for the position of our arrow, the arrow position is managed by Popper.js not our CSS